### DISCUSSION TITLE
Nemo file manager bug with menu bar

### DISCUSSION THREAD

**Discussion by H1manshu-Soni** [prasanthrangan/hyprdots#1427] [General]
![image](https://github.com/prasanthrangan/hyprdots/assets/84933847/c92ee7c2-5d84-4f4c-b7d6-872f2eb6d922)

I dont think this is normal and this is happening in every theme

**Comment by kRHYME7** [comment]
Can you describe what happened?

**Reply by H1manshu-Soni** [reply]
The topbar "FileEditView...." 
There are no spaces which makes it look cramped it's because of new GTK theming
I will include screenshots.


Wallbash mode: theme mode aka disabled
![image](https://github.com/prasanthrangan/hyprdots/assets/84933847/d1358a01-6001-4c4d-9497-a850ac02fcda)


Wallbash mode: auto/dark/light ( the colors might change but the bar remains same ) 
![image](https://github.com/prasanthrangan/hyprdots/assets/84933847/34b92841-c5ae-4aa0-8846-d70739168494)

**Comment by ghost** [comment]
Looks like you're using Wallbash and Nemo file manager is using gtk wallbash theme, which to be honest is currently not that great right now. Disabling wallbash will likely make it look a lot better.

**Reply by H1manshu-Soni** [reply]
Yep but I think it will get better very soon.

**Comment by mahaveergurjar** [comment]
The same this happened with Bluetooth
![image](https://github.com/prasanthrangan/hyprdots/assets/57872034/0bf1c922-aed8-42fa-97cf-df88c26fff53)
![image](https://github.com/prasanthrangan/hyprdots/assets/57872034/e0426195-20f4-433a-803b-c9ec30d8c90d)

**Reply by H1manshu-Soni** [reply]
yes you are right
I just noticed after this comment.

**Comment by kRHYME7** [comment]
Hello!
I think this is already fixed. 

However, can someone point me to the right track of where can I modify the same issue. Planning to make a Dark Red theme but got some issues of the gtk I picked. 


![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/8ee16cef-1c6d-4ff7-a09f-877c688ac1ff)

**Reply by H1manshu-Soni** [reply]
I am not exactly sure but it seems to be have been fixed ( I am on the latest build right now ).

**Reply by kRHYME7** [reply]
Thanks for confirming, I still can't find the specific commit that fixes this. I moved and just used another gtk theme.
